atc/atccmd/command.go

```diff
@@ -89,7 +89,10 @@ type RunCommand struct {
 	ExternalURL flag.URL `long:"external-url" description:"URL used to reach any ATC from the outside world."`
 	PeerURL     flag.URL `long:"peer-url"     description:"URL used to reach this ATC from other ATCs in the cluster."`
 
-	Postgres flag.PostgresConfig `group:"PostgreSQL Configuration" namespace:"postgres"`
+	Postgres            flag.PostgresConfig `group:"PostgreSQL Configuration" namespace:"postgres"`
+	DBKeepAliveIdleTime time.Duration       `long:"db-tcp-keepalive-time" description:"Set net.ipv4.tcp_keepalive_time for DB session. (0 means disable this feature)" default:"0m"`
+	DBKeepAliveCount    int                 `long:"db-tcp-keepalive-probes" description:"Set net.ipv4.tcp_keepalive_probes for DB session. (0 means disable this feature)" default:"0"`
+	DBKeepAliveInterval time.Duration       `long:"db-tcp-keepalive-intvl" description:"Set net.ipv4.tcp_keepalive_intvl for DB session. (0 means disable this feature)" default:"0m"`
```

Comment:
I think it's probably safe to have this on by default, since we already have it on by default in other places.

How about just defaulting to the values from here, since that seems to be working OK for tsa <-> worker?:

https://github.com/concourse/concourse/blob/90a604c940b240a520b284e9a4ba5ad7062c4189/tsa/tsacmd/dialer_unix.go#L18

(Or some other defaults if you've already got opinions - those are pretty arbitrarily chosen.)

Reply:
Sure, `10*time.Second, 3, 5*time.Second` works for me too.